metacat-common-server/src/main/java/com/netflix/metacat/common/server/properties/Definition.java

```diff
@@ -0,0 +1,73 @@
+/*
+ *
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.metacat.common.server.properties;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.netflix.metacat.common.QualifiedName;
+import lombok.NonNull;
+
+import java.util.Set;
+
+/**
+ * Definition metadata related properties.
+ *
+ * @author amajumdar
+ * @since 1.1.1
+ */
+@lombok.Data
+public class Definition {
+
```

Comment:
This class is named as 'Definition'  is strange, which doesn't tell much about itself.

Reply:
Maybe DefinitionMetadataProperties ?

Reply:
It is under properties package and also we have followed this convention for other properties as well. This class refers to property metacat.definition.metadata...